File: springconjava/src/main/java/com/alexandermiss/demo/springconjava/controller/StatusController.java
package com.alexandermiss.demo.springconjava.controller;

import java.util.HashMap;
import java.util.Map;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController()
public class StatusController {

	@GetMapping(value="/status", produces = "application/json")
	public ResponseEntity<Object> status(){
		Map<String, String> resp = new HashMap<>();
		resp.put("status", ":D");
		return ResponseEntity.ok(resp);
	}
}

File: springconkotlin/src/main/kotlin/com/alexandermiss/demo/springconkotlin/controller/StatusController.kt
package com.alexandermiss.demo.springconkotlin.controller

import org.springframework.http.MediaType
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class StatusController {

    @GetMapping(value= ["/status"], produces = [MediaType.APPLICATION_JSON_VALUE])
    fun status() : ResponseEntity<Any> {
        val resp : HashMap<String, String> = HashMap();
        resp["status"] = ":D";
        return ResponseEntity.ok(resp);
    }
}